web/frontend/src/components/base.vue

```diff
@@ -1,21 +1,29 @@
 <script setup lang="ts">
+
 import { grpc } from '@improbable-eng/grpc-web';
 import { ref, onMounted } from 'vue';
-import { computeKeyboardBaseControls, BaseControlHelper } from '../rc/control_helpers';
 import baseApi from '../gen/proto/api/component/base/v1/base_pb.esm';
 import commonApi from '../gen/proto/api/common/v1/common_pb.esm';
-import { toast } from '../lib/toast';
 import { filterResources, type Resource } from '../lib/resource';
 import { displayError } from '../lib/error';
-import KeyboardInput from './keyboard-input.vue';
+import KeyboardInput, { type Keys } from './keyboard-input.vue';
 import { addStream, removeStream } from '../lib/stream';
+import { rcLogConditionally } from '../lib/log';
 import type { ServiceError } from '../gen/proto/stream/v1/stream_pb_service.esm';
 
 interface Props {
   name: string;
   resources: Resource[];
 }
 
+// eslint-disable-next-line no-shadow
```

Comment:
Hmm, for some reason ESLint thinks that we're shadowing another variable with `const enum`s. That's weird, we'll need to investigate more.